README.md

```diff
@@ -15,6 +15,39 @@ for Windows' [Google Group](http://groups.google.com/group/git-for-windows),
 and [contribute bug
 fixes](https://github.com/git-for-windows/git/wiki/How-to-participate).
 
+To build Git for Windows, please either install [Git for Windows'
+SDK](https://gitforwindows.org/#download-sdk), start its `git-bash.exe`, `cd`
+to your Git worktree and run `make`, or open the Git worktree as a folder in
+Visual Studio.
+
+To verify that your build works, use one of the following methods:
+
+- If you want to test the built executables within Git for Windows' SDK,
+  prepend `<worktree>/bin-wrappers` to the `PATH`.
+- Alternatively, run `make install` in the Git worktree.
+- If you need to test this in a full installer, run `sdk build
```

Comment:
I don't have an sdk executable in my bash windows.  Can you elaborate on what this is and where I can get it?

Reply:
It is a Bash alias, not an executable, and when you run `sdk help` in your Git _SDK_ Bash, it will even describe a bit more about its functionality.

Reply:
Right, I should really drive home that the `sdk` alias is only installed in the Git _SDK_ Bash, i.e. you need to call the `git-bash.exe` of your Git for Windows SDK to have it available.